PhotoStoryToBloomConverter/BloomModel/BloomDocument.cs

```diff
@@ -57,9 +58,58 @@ public BloomDocument(PhotoStoryProject project, string bookName)
                     narrationPath = unit.Narration.Path;
 
                 var bloomAudio = new BloomAudio(narrationPath, backgroundPath);
-                _pages.Add(BloomPage.BloomImageOnlyPage(bloomImage, bloomAudio));
+                if (!imageIsCreditsOrCover(Path.Combine(bookDirectoryPath, image.Path)))
+                    _pages.Add(BloomPage.BloomImageOnlyPage(bloomImage, bloomAudio));
             }
+        }
+
+        private readonly string[] creditImages = { "Credit-Beginning.jpg", "Credit-SinEnters.jpg", "Credit-Moses.jpg", "Credit-Ruth.jpg" };
+        private readonly string[] coverImages = { "Cover-Beginning.jpg", "Cover-Lazarus.png" };
+
+        private bool imageIsCreditsOrCover(string imagePath)
+        {
+            foreach (var creditImagePath in creditImages)
+            {
+                if (FileCompare(imagePath, Path.Combine("Credit Images", creditImagePath)))
+                    return true;
+            }
+            foreach (var coverImagePath in coverImages)
+            {
+                if (FileCompare(imagePath, Path.Combine("Cover Images", coverImagePath)))
+                    return true;
+            }
+            return false;
+        }
+
+        private bool FileCompare(string file1, string file2)
```

Comment:
Seems rather brute-force to embed these images and then do actual comparisons. Instead of embedding image files, I think I would only record the checksum (and file size, if performance was a concern) of files of various categories of files.